s2d-oms-mgmt-solution/azuredeploy.json

```diff
@@ -0,0 +1,1847 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "omsLogAnalyticsWorkspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of existing OMS Log Analytics workspace."
+      }
+    },
+    "omsLogAnalyticsWorkspaceLocation": {
+      "type": "string",
+      "allowedValues": [
```

Comment:
is there a reason we have location as a parameter? We generally recommend just using "[resourceGroup().location]"